### PR TITLE
fix: fix receipt too log while typeorm string default is 255

### PR DESCRIPTION
**Pull request by fpChan**

### CODE REVIEW

File: offchain-modules/packages/x/src/db/entity/CkbMint.ts [hunk 1]
```diff
@@ -17,14 +17,14 @@ export class CkbMint {
   @Column()
   amount: string;
 
-  @Column()
+  @Column('varchar', { length: 1024 })
```
Comment (solargatsby) [inline offchain-modules/packages/x/src/db/entity/CkbMint.ts:20]: Is change to text more better？

Comment (solargatsby) [inline offchain-modules/packages/x/src/db/entity/CkbMint.ts:20]: Is change to text more better？

**Comment by huwenchao** [comment]
should analysis max size for every specific chain.
This time we will only change ETH.
The max tx size for CKB is around 500k bytes, so in the extreme case, even TEXT is not big enough.
The max size of both varchar and text  are 65535, but text is slower and can not be indexed.
We can use varchar 10480 for both sudtExtraData and recipientLockscript. It should fit most cases.
Later we will add a sudtExtraDataSizeLimit and recipientAddressSizeLimit limit in config, if the size exceeds the limit, we will log a warn message and ignore it to prevent the service stucking.

File: offchain-modules/packages/x/src/db/entity/CkbMint.ts (hunk 1 above)
Comment (fpChan) [reply offchain-modules/packages/x/src/db/entity/CkbMint.ts]: It has been changed to 10240

Comment (fpChan) [reply offchain-modules/packages/x/src/db/entity/CkbMint.ts]: It has been changed to 10240

**Pull request merged**